### ISSUE TITLE
name 'UserFacingException' is not define

### ISSUE THREAD

**Issue by EchterAgo** [Electron-Cash/Electron-Cash#1761]
Happens when trying to connect a Trezor with outdated firmware. The code is here:

https://github.com/Electron-Cash/Electron-Cash/blob/master/plugins/trezor/clientbase.py#L90

**Comment by cculianu** [comment]
Eek.  That looks like maybe some Electrum code. Electrum has this exception.  Probably the backporter missed a spot.

https://github.com/spesmilo/electrum/blob/e876cb0d931fc5fe9fd7763d15aa59112cfdd357/electrum/util.py#L190

We probably just need to rename that `Exception` and also figure out where it goes if raised (like does the user end up seeing it?).  Or even create that class and make sure the user ends up seeing its message.

**Comment by EchterAgo** [comment]
Yea, I think Electrum uses `UserFacingException` for exceptions that get displayed to the user and should have their text translated. I would favor adding something similar and using that in the future.

**Comment by cculianu** [comment]
Yeah -- but it's work! Ok... maybe after CashFusion is out... or if you feel like tackling it, or anybody else, PRs welcome! :D
